src/esm_runscripts/filedicts.py

```diff
@@ -65,8 +67,70 @@ def __init__(self, full_config, attrs_address):
     def cp(self) -> None:
         pass
 
-    def ln(self) -> None:
-        pass
+    def ln(
+        self, source_path: Union[AnyStr, os.PathLike], destination_path: str
```

Comment:
Why is `AnyStr` needed here as opposed to just regular `str`? Is that a Python version difficulty?

Reply:
I think it is better use this one since it comes from the typing moduel.